src/content/docs/en/upgrade-astro.mdx

```diff
@@ -180,3 +180,24 @@ A major release may also include some non-breaking changes and improvements that
   Once these features are out of their experimental period, they will follow the normal semantic versioning contract.
 
 - **Improvements to the documentation** (e.g. reference and error messages). They are built from source for the `docs` repository. This allows Astro to quickly update docs fixes and improvements in the cases where documentation source content is stored in the main `astro` repository.
+
+### Node.js support and upgrade policies
+
+#### Support
+
+- Astro supports the **latest _extended_ LTS** versions of Node.js.
+- Astro can support odd versions of Node.js.
+
+#### Upgrade
+
+The following rules define when Astro may deprecate, drop, or add support for versions of Node.js:
+
+- Odd versions of Node.js can be deprecated and/or dropped when the next even version of Node.js published. This change can occur in a **minor** release of Astro, after an arbitrary period of time decided by the Astro Core team.
+- Upgrading the minimum **_extended_ LTS** version of Node.js can occur in a **minor** release of Astro.
```

Comment:
By "Upgrading the minimum extended LTS version", does it mean an Astro minor can bump the minimum version only _within the same LTS_ (from the current `18.14.1` to `18.20.0` for example) or can it also bump to the next LTS once the current loses support (from `18` to `20` once support for Node 18 ends in June 2025)?

Reply:
Within the same LTS, as per your example